exporter/elasticsearchexporter/data_stream_router.go

```diff
@@ -29,6 +29,7 @@ const (
 	maxDataStreamBytes       = 100
 	disallowedNamespaceRunes = "\\/*?\"<>| ,#:"
 	disallowedDatasetRunes   = "-\\/*?\"<>| ,#:"
+	encodingFormatAttribute  = "encoding.format"
```

Comment:
We have a set of these consts in `exporter/elasticsearchexporter/internal/elasticsearch/attribute.go` but I think this encodingFormatAttribute is fine here as it is a bit strange to have it under `elasticsearch/attribute` if this is purely for routing. A rename would be nice to align with the other constants though.
```suggestion
	encodingFormatAttributeName  = "encoding.format"
```